mergify_engine/__init__.py

```diff
@@ -12,12 +12,16 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+import cachecontrol
+from cachecontrol.caches import FileCache
```

Comment:
I feel like this is going to be lost on restart right?

Reply:
It's was for my testing, it's going to be redis at this end, I have already prepared a new redis instance in production. We now have 5 servers, so we can't downgrade redis plan anymore.